tests/Integration/ORM/ProxyPHP84Test.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Zenstruck\Foundry\Tests\Integration\ORM;
+
+use PHPUnit\Framework\Attributes\Depends;
+use PHPUnit\Framework\Attributes\RequiresPhp;
+use PHPUnit\Framework\Attributes\Test;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+use Zenstruck\Foundry\Persistence\Proxy\CreatedObjectsTracker;
+use Zenstruck\Foundry\Test\Factories;
+use Zenstruck\Foundry\Tests\Fixture\Factories\Entity\Contact\ContactFactory;
+use Zenstruck\Foundry\Tests\Fixture\Factories\Entity\GenericEntityFactory;
+
+final class ProxyPHP84Test extends WebTestCase
```

Comment:
Let's make sure we test:
1. An object was deleted
2. What happens if you re-proxify over and over